ibm/mas_devops/roles/cp4d_wds/templates/subscription.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    app.kubernetes.io/instance: ibm-watson-discovery-operator-subscription
+    app.kubernetes.io/managed-by: ibm-watson-discovery-operator
+    app.kubernetes.io/name: ibm-watson-discovery-operator-subscription
+  name: ibm-watson-discovery-operator-subscription
+  namespace: "{{ cpdopsnamespace }}"
+spec:
+  channel: "{{ wdschannel is defined | ternary(wdschannel,'v4.0') }}"
```

Comment:
default can be removed here when it is added to the default/main.yml

Reply:
moved to default.yml